charts/devlake/templates/statefulsets.yaml

```diff
@@ -86,11 +86,11 @@ spec:
           volumeMounts:
             - mountPath: /var/lib/mysql
               name: {{ include "devlake.fullname" . }}-mysql-data
-            {{- if ne .Values.option.localtime "" }}
-            - name: {{ include "devlake.fullname" . }}-mysql-localtime
-              mountPath: /etc/localtime
-              readOnly: true
-            {{- end }}
+            # {{- if ne .Values.option.localtime "" }}
+            # - name: {{ include "devlake.fullname" . }}-mysql-localtime
+            #   mountPath: /etc/localtime
+            #   readOnly: true
+            # {{- end }}
```

Comment:
Are you just removing the block without an alternative? If so, the values should be adjusted in consequence.

Reply:
nvm - in the values.yaml obviously